startup/00-startup.py

```diff
@@ -1,17 +1,31 @@
+import sys
+from pathlib import Path
+
+paths = [
+    path
+    for path in Path(
+        "/nsls2/data/sst1/ucal/shared/config/bluesky/collection_packages"
+    ).glob("*")
```

Comment:
Shouldn't this be accomplished with `BS_PYTHONPATH`?

Reply:
This may warrant a broader discussion to develop a standard practice. 

A few beamlines (SST1/2) are developing packages to support their end stations and importing them into the startup files. When some/all of these packages are shared between end stations, it makes sense to have them separate from the startup profile (hence why they are not in `.../'bluesky/collection_profile`). 

The reason they cannot be contained in `BS_PYTHONPATH` is because of issues with combining `-e` and `--prefix` with `pip install`. 
So @danielballan and I discussed and landed here. Packages using a fixed version or external development can land in the environment using `pip install --prefix ...`. Packages that are under continuous development to support the beamline should be part of  the collection_profile or part of collection_packages.

Reply:
At ISS, the repos under active development were put to the shared location and each repo's path was added to `BS_PYTHONPATH` without the need to do `pip install ...`. Maybe that can be a solution?

Reply:
We thought about this initially, but had two concerns: 
1. Each endstation at SST has broken down their configuration into multiple packages, so this gave us a catch all approach in a small block of code. 
2. We wanted a solution that would scale without effort to adding, removing, or refactoring the packages. Changing `BS_PYTHONPATH` would require someone in DSSI running a Satellite role (or extending in the `.bashrc` on a per machine basis). In this case a beamline scientist could add a new analysis package under rapid development and the namespace will automatically be updated.